benchmark/benchwrapper/README.md

```diff
@@ -0,0 +1,10 @@
+# Benchwrapper
+
+A small gRPC wrapper around the spanner client library. This allows the
+benchmarking code to prod at spanner without speaking Python.
+
+## Running
+Run the following commands from python-spanner/ directory.
+```
+export SPANNER_EMULATOR_HOST=localhost:8080
```

Comment:
Is `8080` the port you used while testing? The other client libraries are using `9010`

Reply:
Thanks for pointing it out, Zoe! I've updated this to 9010, we can have it consistent across client libraries.